prompts/2024/english/prompt_2024_01/AnaColon_summary_Bingham_and_Ratcliff_2024

```diff
@@ -0,0 +1,17 @@
+Complex multicellularity has evolved five separate times in eukaryotes but has never evolved in prokaryotes. Complex multicellularity has been defined as a large organism composed of many cells that lives for an extensive period of time. Some prokaryotes are multicellular but they do not have the morphologies of complex multicellular organisms. 
+
+This paper proposes a new hypothesis as to why prokaryotes lack complex multicellularity compared to eukaryotes. When bottle neck events causing a reduced efficient population size cells are divided into groups. In response to this eukaryotes will undergo genomic expansion while prokaryotes will undergo genomic deletion. These differences in responses could be an explanation as to why eukaryotes have been able to evolve complex multicellularity and prokaryotes have not. 
+
+Eukaryotic have transposons and introns that facilitate genomic expansion through replication. Prokaryotes tend to have deletion bias where they will delete parts of their genome. These subcellular mechanisms influence their response to bottleneck effects. 
+
+With an expanded genome, eukaryotes have more potential for the emergence of new genes. This would facilitate their evolution after experiencing a bottleneck effect with a reduced population size. Prokaryotes would lose genetic material which would cause less genetic diversity as they try to evolve. Less genomic material leads to less opportunity for evolutionary events to occur. 
+
+When comparing complex multicellular genomes to unicellular relatives in fungal lineages, the genomes for complex organism are twice as big. This includes protein-encoding regions and non-coding regions that can serve as evolutionary regions for gene regulation. 
+
+Cyanobacteria have a large genome, however, they have a high frequency of pseudogenes that would result from gene deletion. This supports the hypothesis that prokaryotes favor gene deletion compared to gene replication. 
```

Comment:
'Cyanobacteria have a large genome' - relatively speaking. Large for a bacterium; still not large compared to eukaryotes. For example, the human genome is 6.37 Gbp but the largest cyanobacterial genome in this paper's figure 1A is ~13 Mbp (500 times fewer A's, T's, C's, and G's)